qemu/tests/transfer_file_over_ipv6.py

```diff
@@ -67,16 +77,27 @@ def get_file_md5sum(file_name, session, timeout):
         if params.get("os_type") == "linux":
             inet_name[vm] = utils_net.get_linux_ifname(sessions[vm],
                                                        vm.get_mac_address())
-        addresses[vm] = utils_net.get_guest_ip_addr(
-            sessions[vm],
-            vm.get_mac_address(),
-            params.get("os_type"),
-            ip_version="ipv6",
-            linklocal=link_local_ipv6_addr)
-        if link_local_ipv6_addr is False and addresses[vm] is None:
-            test.cancel("Your guest can not get remote IPv6 address.")
-        error_context.context("Get ipv6 address of %s: %s" % (vm.name, addresses[vm]),
-                              test.log.info)
+        if nettype == "vdpa":
+            guestid = random.randint(1, 30)
```

Comment:
There is a probability that both the host and the guest will get the number 30.

Reply:
> There is a probability that both the host and the guest will get the number 30.

Updated, please help review it again, thanks a lot.